README.md

```diff
@@ -1,6 +1,10 @@
 # Installation
 
     git clone https://github.com/tswr/.vimrc.git ~/vim-tswr/
+    bash ~/vim-tswr/setup.sh
```

Comment:
I think It will not work because   https://github.com/tswr/.vimrc/blob/master/setup.sh returns html-file (at least for me)